llvm/test/SYCLLowerIR/esimd_lower_intrins.ll

```diff
@@ -14,7 +14,9 @@ target triple = "spir64-unknown-unknown-sycldevice"
 @vg = dso_local global %"cm::gen::simd<int, 16>" zeroinitializer, align 64 #0
 @vc = dso_local addrspace(1) global <32 x i32> zeroinitializer
 
-define dso_local spir_func <32 x i32>  @FUNC_1() !sycl_explicit_simd !1 {
+; LowerESIMD pass should process every function, 
+; !sycl_explicit_simd metadata is not necessary.
+define dso_local spir_func <32 x i32>  @FUNC_1() {
```

Comment:
There are a number of other functions with this metadata used for LowerESIMD testing: 
- right below in this test
- in other tests such as llvm/test/SYCLLowerIR/scalar_fptoui.ll

they should be corrected as well then, I think.

Reply:
Ok, will do.

Reply:
@kbobrovs , done.